lib/htty/cli/display.rb

```diff
@@ -153,11 +153,12 @@ def show_response(response)
     end
     print ' '
     cookies_asterisk = response.cookies.empty? ? '' : strong('*')
-    body_length = response.body.to_s.length
-    body_size   = body_length.zero? ? 'empty' : "#{body_length}-character"
+    body_length      = response.body.to_s.length
+    body_size        = body_length.zero? ? 'empty' : "#{body_length}-character"
+    response_time    = (response.time * 1000).round(2)
     puts([description,
           pluralize('header', response.headers.length) + cookies_asterisk,
-          "#{body_size} body"].join(' -- '))
+          "#{body_size} body", "#{response_time} ms"].join(' -- '))
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.